cwl/workflow/split_docker/gatk-split-interval-list.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
```

Comment:
duplicate file under docker and under the scripts directory - there should be only one